sites/partners/src/listings/PaperListingForm/sections/BuildingDetails.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react"
+import { useFormContext } from "react-hook-form"
+import {
+  t,
+  GridSection,
+  GridCell,
+  Field,
+  ViewItem,
+  Select,
+  stateKeys,
+} from "@bloom-housing/ui-components"
+
+const BuildingDetails = () => {
+  const formMethods = useFormContext()
+
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register } = formMethods
+
+  return (
+    <GridSection grid={false} columns={3} separator>
+      <span className="form-section__title">{t("listings.sections.buildingDetailsTitle")}</span>
+      <span className="form-section__description">
+        {t("listings.sections.buildingDetailsSubtitle")}
+      </span>
+
+      <GridSection columns={3}>
+        <GridCell span={2}>
+          <Field
+            label={t("application.contact.streetAddress")}
+            name={"buildingAddress.street"}
```

Comment:
It's fine either way, but I'd suggest using the `prop="string"` instead of `prop={"string"}` style unless there's something dynamic/computed going on. (just a little easier to read)